en/getting-started.md

```diff
@@ -1,10 +1,106 @@
+
+{% hint style="info" %}
+The usage of a text-based file format has some advantages:
+
+- The file is "human readable" and editable with every text editor
+- the text format allows for an easy tracking of changes with every common version control protocol (e.g., git)
+- and finally: the format is dedicated for the usage with LaTeX; so you do not need to convert it to any other format but you can just directly link to your JabRef library
+
+{% endhint %}
+
+To create a new library, just select the "New library" menu item in the "File" menu:
+
+![Creating a new library](.gitbook/assets/getting-started-new-library.png)
+
+The main screen is now showing an empty "entry table" we will now start to fill with some entries.
+
+## Adding of a new entry manually
+
+To add a new entry select the menu bar entry "Library" -> "New entry", click on the icon in the icon bar, or just hit CTRL-N.
+
+This opens a dialog where you can select the type of reference you want to store. By default all entry types defined by the Bibtex format are available:
+
+![Screenshot of "new entry" dialog](.gitbook/assets/getting-started-new-entry.png)
+
+For our running example we will select "Article".
+
+After clicking on the "Article" button, the dialog closes and the so called "Entry Editor" is opened for the newly created entry:
+
+![Main window now showing the entry editor](.gitbook/assets/getting-started-entry-editor.png)
+
+The most important information about the references to be added can now be entered in the "Required Fields" tab.
+"Author", "Title", "Journal", and "Year" should be self-explanatory - however, a "citationkey", might not be familiar to you.
+Basically, the idea of the "citationkey" (formerly called "bibtexkey") is coming from working with Bibtex, where it is necessary to have an unique identifier for each entry. This allows for referencing within a document you might be creating using the stored information in your library. Moreover, also within JabRef this "key" is used for example for cross-references to other related entries or to determine file names for full-text references.
+
+The key usually follows a global pattern and can be easily created automatically by clicking on the "generate" button next to the field.
+
 {% hint style="info" %}
-Coming soon!
+The default key pattern is `[auth][year]`, which means that Author information is followed by the year of the publication, resulting in the example in `Turing1950`. However, the key pattern is customizable to your needs. See [Configuration](https://docs.jabref.org/setup) > ["Customize the citation key generator"](https://docs.jabref.org/setup/bibtexkeypatterns) for more details.
 {% endhint %}
 
-Desktop interface \(basic description of the user interface elements and what they do\)
+After entering some information, you can see on the right side of the entry editor a preview of the bibliographic data:
+
+![Added information for new entry](.gitbook/assets/getting-started-filled-entry-editor.png)
 
```

Comment:
Point them to https://docs.jabref.org/collect for further possibilities to add entries.

Reply:
Done - hopefully in the right syntax - needs to be confirmed after pushing to master in "Gitbook" view.